### PR TITLE
W-13915584 MQ: FIFO and Prefetch again

### PR DESCRIPTION
**Pull request by hannanelson**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

image

image

### CODE REVIEW

**Comment by somal-vora** [comment]
@yunmei-zhang Please review too.

**Review by somal-vora — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/mq-queues.adoc [hunk 1]
```diff
@@ -319,23 +319,23 @@ However, Anypoint MQ cannot guarantee strict message ordering in this case.
...
+* FIFO queues are limited to 300 TPS (transactions per second).
 +
 However, if you batch ten messages per read and write operation (maximum) using the API, FIFO queues can support up to 3,000 TPS.
 +
 //Shared TPS failed
 include::partial$mq-faq.adoc[tag=failedTPS]
-* Can't be bound to a message exchange.
+* You can't bind FIFO queues to a message exchange.
+* In some configurations when using the IMMEDIATE acknowledgment mode in the Subscriber operation, FIFO queues don't process messages in order.
+* Using `prefetch` mode with FIFO queues is not recommended.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/mq-queues.adoc]: According to our style guide, we should not "recommend":
Before using `prefetch` mode with FIFO queues, see xref:anypoint-mq-connector::anypoint-mq-listener.adoc#fifo-and-prefetch[FIFO Queues and Prefetch Mode].

**Review by Crispy-Salesforce — DISMISSED** [review]

File: modules/ROOT/pages/mq-queues.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/mq-queues.adoc]: Yes. Good suggestion. Thanks.

**Review by Crispy-Salesforce — APPROVED** [review]

**Comment by yunmei-zhang** [comment]
LGTM

**Pull request merged**